### PR TITLE
make `get_robustcov_results` calls from `Results.__init__`

### PR DESCRIPTION
**Pull request by jbrockmendel**
not from `Model.fit`

### CODE REVIEW

File: statsmodels/base/model.py [hunk 1]
```diff
@@ -1027,13 +1027,11 @@ def __init__(self, model, params, normalized_cov_params=None, scale=1.,
...
                                  'specified.'}
             else:
-                from statsmodels.base.covtype import get_robustcov_results
                 if cov_kwds is None:
                     cov_kwds = {}
                 use_t = self.use_t
-                # TODO: we shouldn't need use_t in get_robustcov_results
-                get_robustcov_results(self, cov_type=cov_type, use_self=True,
-                                           use_t=use_t, **cov_kwds)
+                self._get_robustcov_results(cov_type=cov_type, use_self=True,
```
Comment (josef-pkt) [inline statsmodels/base/model.py]: this might be questionable
I think it is supposed to use only the standalone function

**Comment by josef-pkt** [comment]
Most likely this change is good, after checking some things.

File: statsmodels/base/model.py (hunk 1 above)
Comment (jbrockmendel) [reply statsmodels/base/model.py]: Take a look at the method.  It's the same.

**Comment by jbrockmendel** [comment]
Closes #4025

**Comment by josef-pkt** [comment]
This pull request introduces 1 alert - [view on lgtm.com](https://lgtm.com/projects/g/statsmodels/statsmodels/rev/pr-b882ac45705774f529fb36e44b9e55919e74d8db)

**new alerts:**

* 1 for __init__ method calls overridden method

---

*Comment posted by [lgtm.com](https://lgtm.com)*

**Comment by josef-pkt** [comment]
I need to think about this, especially the interaction with transparams and the second commit here (where cov_type is computed twice)

postponed to 0.10

**Comment by jbrockmendel** [comment]
> I need to think about this, especially the interaction with transparams and the second commit here (where cov_type is computed twice)

I'll have to re-familiarize myself with this to see if I can say anything reassuring.

Here, have three positive-reinforcement points for maintenance work today.

**Pull request closed without merging**